pkg/webhook/clusterresourceoverride/clusterresourceoverride_validating_webhook.go

```diff
@@ -30,7 +30,7 @@ var (
 
 type clusterResourceOverrideValidator struct {
 	client  client.Client
-	decoder *admission.Decoder
+	decoder webhook.AdmissionDecoder
```

Comment:
I wonder why is this related to Kruise or controller-runtime?

Reply:
it's related to the controller-runtime package. The latest version of Decoder is an interface.

// AdmissionDecoder knows how to decode objects from admission requests.
type AdmissionDecoder = admission.Decoder

https://github.com/kubernetes-sigs/controller-runtime/blob/1f5b39fa59d15fae78e521c9c9f2acabbbb3ea17/pkg/webhook/admission/decode.go#L29